site/en/blog/storage-partitioning-deprecation-trial/index.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: "layouts/blog-post.njk"
+title: "Participate in Deprecation Trial for unpartitioned third-party storage, Service Workers, and Communication APIs"
```

Comment:
Also, I am not exactly sure how the title should be capitalized, but this seems a little wonky (my bad)? Maybe "Participate in Deprecation Trial for Unpartitioned Third-Party Storage, Service Workers, and Communication APIs"